connectors/webservice/classes/external/receive_message.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace mbconnector_webservice\external;
+
+use external_api;
+use external_value;
+use moodle_exception;
+use tool_messagebroker\message\received_message;
+use tool_messagebroker\message_processor;
+use tool_messagebroker\receiver\processing_style;
+
+class receive_message extends external_api {
+
+    public static function execute(string $topic, string $body) {
+        // TODO: authorisation
```

Comment:
@darrencocco I'm not sure if anything is actually needed here? The service is behind a cap check - is that enough or do we need something else?

Reply:
Last I checked the capabilities specified in services.php are just informational and don't actually get used to enforce authorisation checks.

Reply:
Wow, TIL. It's [even documented](https://moodledev.io/docs/apis/subsystems/external/writing-a-service)!

![image](https://github.com/darrencocco/moodle-tool_messagebroker/assets/89116254/5f981064-5b6f-41e5-baec-e41bcf35846d)

Pushed a commit to add an explicit cap check.